### PR TITLE
Enable resource watch for e2e-azure-manual-oidc

### PR DESCRIPTION
**Pull request by xueqzhan**

### CODE REVIEW

**Comment by xueqzhan** [comment]
pj-rehearse periodic-ci-openshift-cloud-credential-operator-release-4.15-periodics-e2e-azure-manual-oidc

File: ci-operator/jobs/ostreedev/ostree/ostreedev-ostree-main-presubmits.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ presubmits:
     branches:
     - ^main$
     - ^main-
-    cluster: build02
```
Comment (deads2k) [inline ci-operator/jobs/ostreedev/ostree/ostreedev-ostree-main-presubmits.yaml:8]: weird.  bad merge?

**Comment by xueqzhan** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.15-e2e-azure-manual-oidc

**Comment by jstuever** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-release-4.15-e2e-azure-manual-oidc

**Pull request merged**